xampp/README.md

```diff
@@ -0,0 +1,17 @@
+# Installing xampp
+
+This example was chosen because it installs to a location unsupported by rpm-ostree(/opt/)
+
+In this example, we:
+
+- store the installer in var/
```

Comment:
This part I think is a bit confusing; it'd be better to use `/tmp`.

Reply:
That makes more sense, agreed!